src/tools/miri/tests/pass/intrinsics-integer.rs

```diff
@@ -1,12 +1,5 @@
-// Copyright 2012-2014 The Rust Project Developers. See the COPYRIGHT
-// file at the top-level directory of this distribution and at
-// http://rust-lang.org/COPYRIGHT.
-//
-// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
-// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
-// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
-// option. This file may not be copied, modified, or distributed
-// except according to those terms.
```

Comment:
FWIW these are copies of old rust source files, that had their license headers removed in rustc at some point but that change was never synced to Miri. So now this 'fork' has the SPDX header but the original file that still exists in rustc does not... it'd be more consistent to just remove the license header here without adding SPDX headers. But either way is fine for me.